integrations/elevenlabs.md

```diff
@@ -0,0 +1,185 @@
+---
+layout: integration
+name: Elevenlabs
+description: ElevenLabs Text-to-Speech components for Haystack.
+authors:
+    - name: Andy
+      socials:
+        github: andychert
+        twitter: andychert
+pypi: https://pypi.org/project/elevenlabs-haystack/
+repo: https://github.com/andychert/elevenlabs-haystack
+type: Model Provider
+report_issue: https://github.com/andychert/elevenlabs-haystack/issues
+logo: /logos/elevenlabs.png
```

Comment:
We'll be using this logo with a dark background so is there a white/light colored version of the logo?

Reply:
Added a white version of the logo.